hm_pyhelper/hardware_definitions.py

```diff
@@ -473,9 +462,44 @@ def is_raspberry_pi() -> bool:
         'CONTAINS_FCC_IDS': [],
         'IC_IDS': [],
         'CONTAINS_IC_IDS': []
-        }
+        },
+
+    # COTX X3 Hotspot
+    'cotx3-v1': {
+        'FRIENDLY': 'COTX X3',
+        'SPIBUS': 'spidev0.0',  # There is a CSN1 pin which is connected to GPIO6 (HAT Pin 31)
+        'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'RESET': 22,
+        'MAC': 'eth0',
+        'STATUS': 21,  # Stub. There is no status LED on X3. I2C-3 is used for display
+                       # communication (HAT pins 5,7)
+        'BUTTON': 23,
+        'ECCOB': True,
+        'TYPE': 'Full',
+        'CELLULAR': False,  # There is a 4G option on the HAT board.
+        'FCC_IDS': [],
+        'CONTAINS_FCC_IDS': [],
+        'IC_IDS': [],
+        'CONTAINS_IC_IDS': []
+        },
 }
 
+# Note: Maintain old names for backward compatibility, should be removed at some
+# point of time.
+variant_definitions['NEBHNT-IN1'] = variant_definitions['nebra-indoor1']
+variant_definitions['NEBHNT-OUT1'] = variant_definitions['nebra-outdoor1']
+variant_definitions['NEBHNT-LGT-ZS'] = variant_definitions['nebra-light1']
+variant_definitions['NEBHNT-LGT-RADXA'] = variant_definitions['nebra-light2']
+variant_definitions['NEBHNT-HHRK4'] = variant_definitions['nebra-indoor2']
+variant_definitions['NEBHNT-HHRK4-OUT'] = variant_definitions['nebra-outdoor2']
+variant_definitions['COMP-RAKHM'] = variant_definitions['rak-v1']
```

Comment:
This is confusing because we don't support rakv1. We only support rakv1.5, v2 and mntd ones

Reply:
to resolve these, I have added supported models key in https://github.com/NebraLtd/hm-pyhelper/pull/173 as I have adding models etc there.